PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -962,10 +967,19 @@ struct HfTrackIndexSkimCreator {
   std::array<std::vector<double>, kN3ProngDecays> pTBins3Prong;
 
   using SelectedCollisions = soa::Filtered<soa::Join<aod::Collisions, aod::HfSelCollision>>;
+  using TracksWithDCA = soa::Join<aod::BigTracks, aod::TracksDCA>;
   using TracksWithPVRefitAndDCA = soa::Join<aod::BigTracks, aod::TracksDCA, aod::HfPvRefitTrack>;
 
+  // filter collisions
   Filter filterSelectCollisions = (aod::hf_sel_collision::whyRejectColl == 0);
-  Filter filterSelectedTrackIds = aod::hf_sel_track::isSelProng > 0;
+
+  // define slice of track indices per collisions
+  Preslice<TracksWithPVRefitAndDCA> tracksPerCollision = aod::track::collisionId; // needed for PV refit
+
+  // filter track indices
+  Filter filterSelectTrackIds = (aod::hf_sel_track::isSelProng > 0);
+  using FilteredTrackAssocSel = soa::Filtered<soa::Join<aod::TrackAssoc, aod::HfSelTrack>>;
```

Comment:
Move `using` up next to the other `using` declarations.

Reply:
Done